### PR TITLE
Home page list

### PR DESCRIPTION
**Pull request by medsonmoombe**
## The following were implemented in this project 

- Add the home page and added movies cards
- modify the footer and the navigation bar
- Added a module folder and a  JavaScript  file which contains a function that renders the content dynamically

### CODE REVIEW

File: src/modules/showMovies.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    ratings: 3,
+    title: 'Hard Target',
+  },
+  {
+    image: '/images/movie1.jpg',
+    ratings: 3,
+    title: 'Hard Target',
+  },
+  {
+    image: '/images/movie1.jpg',
```
Comment (yayner2002) [inline src/modules/showMovies.js:23]: Great job so far. But according to the requirement please use the data that is coming from the API to dynamically render those `images `and `titles `of the movie

**Review by yayner2002 — CHANGES_REQUESTED** [review]
Hi, @medsonmoombe  Great job so far. But there are some issues that you have to work on to go to the next exercise 👍

**Comment by medsonmoombe** [comment]
## The following requested changes were implemented
- data  coming from the API dynamically rendered 
-

**Review by yayner2002 — APPROVED** [review]
- Hi, @medsonmoombe  Great job so far. you have done a great job. But here are a few optional issues you have to work on to go to the next project.

File: src/modules/getdata.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+//             <i class="fa-solid like-btn fa-star"></i>
+//             <p id="rateCounts">${movie.ratings} likes</p>
+//         </div>
+//         <h2 class="movie-title">${movie.title}</h2>
+//         <button class="movie-comment" id="comment-btn">Comments</button>
+//         <button class="movie-reservation" id="reservation-btn">Reservations</button>
+//         </div>
+//         </div>`;
+//     movieDisplay.innerHTML += movieTag;
+//     main.appendChild(movieDisplay);
```
[the remark this answers is not in the document]
Comment (yayner2002) [reply src/modules/getdata.js:28]: Kindly remove unused code from your codebase

**Review by yayner2002 — APPROVED** [review]

File: src/modules/showMovies.js [hunk 3]
```diff
@@ -1,67 +1,45 @@
...
+    const likeCount = document.createElement('p');
+    likeCount.setAttribute('class', 'rateCounts');
+    likeCount.setAttribute('Id', `${data[i].id}`);
+    likeCount.textContent = '0 likes';
+    stats.append(like, likeCount);
+    movieRating.appendChild(stats);
+    movieDescript.append(title, movieRating);
+    const commentBtn = document.createElement('button');
+    commentBtn.classList.add('movie-comment');
+    commentBtn.setAttribute('movie-Id', `${data[i].id}`);
```
[the remark this answers is not in the document]
Comment (medsonmoombe) [reply src/modules/showMovies.js:34]: Alright @yayner2002  thanks for the review

**Pull request merged**